### ISSUE TITLE
fatal error: implicit conversion changes signedness with cython >= 0.28

### ISSUE THREAD

**Issue by cbenhagen** [python-llfuse/python-llfuse#21]
Tested with python 2.7.15 and 3.7.1 on macOS 10.14

```
python setup.py build_ext --inplace
found MANIFEST.in, running in developer mode
running build_ext
building 'llfuse' extension
creating build
creating build/temp.macosx-10.9-x86_64-3.7
creating build/temp.macosx-10.9-x86_64-3.7/src
gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -arch x86_64 -g -I/Library/Frameworks/Python.framework/Versions/3.7/include/python3.7m -c src/llfuse.c -o build/temp.macosx-10.9-x86_64-3.7/src/llfuse.o -D_FILE_OFFSET_BITS=64 -D_DARWIN_USE_64_BIT_INODE -I/usr/local/include/osxfuse/fuse -DFUSE_USE_VERSION=29 -Wall -Wextra -Wconversion -Wsign-compare -DLLFUSE_VERSION="1.3.5" -Wno-unused-function -Wno-implicit-fallthrough -Wno-unused-parameter -Werror -Wfatal-errors -Wno-error=unreachable-code -Werror=conversion -Werror=sign-compare
src/llfuse.c:49425:3: fatal error: implicit conversion changes signedness: 'Py_ssize_t' (aka 'long') to 'unsigned long' [-Wsign-conversion]
  __Pxy_PyFrame_Initialize_Offsets();
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/llfuse.c:1132:60: note: expanded from macro '__Pxy_PyFrame_Initialize_Offsets'
     (void)(__pyx_pyframe_localsplus_offset = PyFrame_Type.tp_basicsize - Py_MEMBER_SIZE(PyFrameObject, f_localsplus)))
                                              ~~~~~~~~~~~~~^~~~~~~~~~~~ ~
1 error generated.
error: command 'gcc' failed with exit status 1
```

**Comment by Nikratio** [comment]
Thanks for the report! This looks like an issue in Cython - it would be great if you could report it upstream. (note that this is only an error when building python-llfuse in developer mode from Git, the release tarballs are not affected).

**Comment by JustAnotherArchivist** [comment]
I'm experiencing the same thing with Cython 0.29 and pyfuse3. Filed upstream as cython/cython#2729.

**Comment by cbenhagen** [comment]
Fixed in Cython 0.29.1
